src/controllers/RepostController.ts

```diff
@@ -0,0 +1,142 @@
+            );
+            if (!postInfo) throw new Error(`Job post cannot be found.`);
+            await this.createSpecificJobPost(
+                job,
+                jobID,
+                jobInfo,
+                postInfo,
+                boardToPost,
+                page
+            );
+            const deleteSpecificPost = await deleteSpecificPostInteractive(
+                this.config,
+                jobPost,
+                jobInfo,
+                postInfo
+            );
+            if (deleteSpecificPost) {
+                await page.reload();
+                console.log(green("✔"), "Old job post deleted.");
+            }
+        } else {
+            if (!postID)
+                throw new UserError(`Job post ID argument is missing.`);
+            this.spinner.start(`Fetching the job information.`);
+            jobID = await job.getJobIDFromPost(postID);
+            jobInfo = await job.getJobData(jobID);
+            if (!jobInfo.posts.length)
+                throw new Error(
+                    `Job posts cannot be found for ${jobInfo.name}.`
+                );
+            this.spinner.succeed();
+
+            const boardToPost = await job.getBoardToPost();
+
+            const postInfo = jobInfo.posts.find((post) => post.id === postID);
+            if (!postInfo) throw new Error(`Job post cannot be found.`);
+            const jobPost = await this.createSpecificJobPost(
+                job,
+                jobID,
+                jobInfo,
+                postInfo,
+                boardToPost,
+                page
+            );
+            const referrer = joinURL(
+                this.config.greenhouseUrl,
+                `/plans/${jobInfo.id}/jobapp`
+            );
+            await jobPost.deletePost(postInfo, referrer);
+            await page.reload();
+            console.log(green("✔"), "Old job post deleted.");
+        }
+
+        console.log("Happy hiring!");
+
+        browser.close();
+    }
+
+    private async createSpecificJobPost(
+        job: Job,
+        jobID: number,
+        jobInfo: JobInfo,
+        postInfo: PostInfo,
+        boardToPost: JobBoard,
+        page: Puppeteer.Page
+    ): Promise<JobPost> {
+        const jobPost = new JobPost(page, this.config);
+        // Remove brackets from the location name
+        const postLocation = postInfo.location.replace(/^\(|\)$/g, "");
+        // Duplicate job post in the same location
+        await jobPost.duplicate(postInfo, postLocation, boardToPost);
+        // Mark all newly added job posts as live
+        await job.markAsLive(jobID, jobInfo.posts, boardToPost);
+        console.log(
+            green("✔"),
+            `New job post for ${postInfo.name} of ${jobInfo.name} is created in ${postLocation}`
+        );
+        // Mark old job post as offline
+        await jobPost.setStatus(postInfo, "offline", boardToPost);
+        console.log(green("✔"), "Old job post marked as offline.");
```

Comment:
Saying that it's marked as offline could be skipped, as the next step is deleting it entirely. Looks more like an implementation detail rather than something that the user should know

Reply:
I've removed action this completely - posts should just be deleted.